### ISSUE TITLE
Does not work well with `html { scroll-behavior : smooth; }`

### ISSUE THREAD

**Issue by sujan-s** [caiofsouza/vue-backtotop#12]
The scrolling back to top becomes extremely slow if you have the following rule in your CSS—
`html { scroll-behavior : smooth; }`.

It takes about four seconds to scroll back to the top with this CSS rule.

**Comment by caiofsouza** [comment]
Hey @sujan-s,
Thank for the feedback and sorry the delay for answer!

Adding `html { scroll-behavior : smooth; }` cause some delay on scroll. I'll check this and try to find some solution.

Feel free to create a pull request if you has some idea.

**Comment by TienTruong9x** [comment]
Yeah! I have the same problem. It's been 3 years but author not fix them  yet :)))))))
